scripts/linux/installPrereqs.sh

```diff
@@ -1,9 +1,6 @@
 #!/bin/bash
 
-# Installs the pre-reqs (currently only libsnappy) on the machine.
-
-echo Install Libsnappy
 sudo apt-get update
-sudo apt-get install -y libsnappy1v5
+sudo apt-get install -y libsnappy1v5 libc6-dev
```

Comment:
I feel like this script should really be part of the test agent image creation process than having it run every iteration.

Reply:
Yeah I think I agree. `installPrereqs.sh` has been around forever though, so a little more care would need to be taken before we remove it.